Zend/zend_objects_API.h

```diff
@@ -137,5 +137,20 @@ static inline zend_property_info *zend_get_typed_property_info_for_slot(zend_obj
 	return NULL;
 }
 
+static inline bool zend_check_method_accessible(const zend_function *fn, const zend_class_entry *scope)
+{
+	if (
+		!(fn->common.fn_flags & ZEND_ACC_PUBLIC)
+		&& fn->common.scope != scope
+		&& (
+			UNEXPECTED(fn->common.fn_flags & ZEND_ACC_PRIVATE)
+			|| UNEXPECTED(!zend_check_protected(zend_get_function_root_class(fn), scope))
```

Comment:
Nit: I know we don't have guidelines, but your formatting diverges quite significantly from the majority of code.

Reply:
Fair enough. I'll adjust this to what I believe is the code style once CI ran to success.